src/PKSim.Presentation/DTO/Mappers/PKSimPathToPathElementsMapper.cs

```diff
@@ -139,16 +139,26 @@ private void adjustDisplayPathForQuantity(PathElements pathElements, IQuantity q
          else if (quantity.IsNamed(PLASMA_UNBOUND))
             adjustDisplayPathForContainerObserver(pathElements, quantity);
 
+         else if (quantity.Name.StartsWith(TOTAL_FRACTION_OF_DOSE))
+            adjustDisplayPathForTotalFractionOfDose(pathElements, quantity);
+
+         //Container observers directly in a container located under organism
          else if (!pathElements.Contains(PathElement.BottomCompartment))
             adjustDisplayPathForContainerObserver(pathElements, quantity);
 
          else if (quantity.IsAnImplementationOf<IObserver>())
             adjustDisplayNameForMoleculeObserver(pathElements, quantity.DowncastTo<IObserver>());
       }
 
+      private void adjustDisplayPathForTotalFractionOfDose(PathElements pathElements, IQuantity quantity)
+      {
+         pathElements[PathElement.Container] = pathElements[PathElement.TopContainer];
+         pathElements[PathElement.TopContainer] = new PathElementDTO();
```

Comment:
I am resetting this guy only so that we do not have TopContainer == Container. Does not have to be
This is with the reset
![image](https://user-images.githubusercontent.com/1041237/55837341-19520500-5aef-11e9-89b5-451f8c6b8709.png)


This is without the reset
![image](https://user-images.githubusercontent.com/1041237/55837373-3090f280-5aef-11e9-8bae-d6db34e3e46d.png)


It's maybe better to have Organism and Organism (e.g no reset) for consistency purpose so that you can group by organism if you want

Thoughts?

Need to add some tests and this is good !

Reply:
@Yuri05 Thoughts? Or better idea for the display?